scripts/python-skywater-pdk/skywater_pdk/cell_list.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
```

Comment:
Maybe we should create a `skywater_pdk/cells` directory and this could be `skywater_pdk/cells/list.py`?

Reply:
fixed